src/Frontend/test/drivers/vitest/driver.ts

```diff
@@ -31,10 +31,9 @@ function makeDriver() {
       return factory({ driver: this });
     },
     disposeApp() {
-      if (!app) {
-        throw new Error("App is not mounted, make your sure to await driver.goTo().");
```

Comment:
@cquirosj [this](https://github.com/Particular/ServicePulse/pull/1909/files?diff=unified#diff-edc4597bf9a577eaf7c535af22acd31e0a8ed0b597acced41339e61289179423R34-R36) change, conflicts, with my change here.
In my tests I found it simpler to not having to deal with navigation and ensuring the whole app is running, hence if you look at some of the tests here, they have less setup.

Reply:
@johnsimons, if you wish to re-use the preconditions functions from the acceptance tests setup, there is no need to use the custom "Test" function to have an instance of the driver. The component tests can create their own driver instance, as demonstrated in the following example where:
- A component test makes use of mocked responses by both:
  - Re-using the pre-defined mocked responses from the acceptance tests (optional)
  - And by setting up the mocked responses directly

https://github.com/Particular/ServicePulse/pull/1916/files#diff-d065d469d8fd3e77aaf7b6e797e3b3e08aa883643c2783a3119048559e0fa3db


By the way, this [PR](https://github.com/Particular/ServicePulse/pull/1916) demonstrates how to test the same scenarios at the view component level vs at the acceptance test level